public_html/wp-content/mu-plugins/blocks/source/blocks/schedule/edit.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * WordPress dependencies
+ */
+import { createContext } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useScheduleData } from './data';
+import { ScheduleGrid } from './schedule-grid';
+import InspectorControls from './inspector-controls';
+import { NoContent } from '../../components/';
+import './edit.scss';
+
+export const ScheduleGridContext = createContext();
```

Comment:
If you pull this out into its own file, you can import it here & in front-end without also importing all the rest of the edit-related dependencies into front-end. Right now I think this is causing a bunch of things to get imported into the front-end build— you can see that `block-editor` is in the asset manifest, and that shouldn't be necessary on the front end.

Reply:
d934e474 only had a marginal impact, but moving it makes sense logically, regardless 👍 

It looks like this accounted for a lot of the size:

`import { NoContent } from '../../components/';`

Fixed in 951a0a09. The front end file is down to 25k now :)